server/load.js

```diff
@@ -134,7 +137,15 @@ exports.handleLoad = function(req, res, app, format) {
           filename += '/';
         }
 
-        var list = buildDirList(absfile, fs.readdirSync(absfile).sort());
+        var files = fs.readdirSync(absfile);
```

Comment:
I'd prefer if you use the DirCache object to read the directory (and apply prefix filtering if requested), modifying DirCache to handle this case correctly.   It would make sense to cache share.pencilcode.net the same way that the root user listing is cached.

The DirCache object will parallelize the stat calls, and so it's a better way to handle directories generally, though beyond share and the root listing, we probably don't want to actually cache DirCache objects.